### PR TITLE
remove cloudmonkey and add some libraries

### PR DESCRIPTION
**Pull request by KazuCocoa**
- remove `CloudMonkey` because it was acquired by Oracle and closed
- add Kotlin test framework
- add security tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ Of course it sometimes includes google's official stuff.
 - [AndroidJUnitRunner](https://developer.android.com/tools/testing-support-library/index.html#AndroidJUnitRunner)
 - [Robolectric](https://github.com/robolectric/robolectric)
 - [RoboSpock](https://github.com/robospock/RoboSpock)
- 
+- [Spec](https://github.com/JetBrains/spek) (Kotlin)
```
Comment (hotchemi) [inline README.md:15]: Spek?

**Comment by hotchemi** [comment]
> remove CloudMonkey because it was acquired by Oracle and closed

Wow!

> Spek

I'm not sure it is specifically for Android. Thoughts?

**Comment by KazuCocoa** [comment]
Like JUnit for Kotlin :)
My co-worker used it with him private android app before. ( But it is a bit unstable. )

So, I set it the same section with JUnit.

**Comment by hotchemi** [comment]
ic. Yeah we can use any jvm based testing framework for a android **unit** test.

Btw, could u check it? https://github.com/hotchemi/awesome-android-testing/pull/10#discussion_r51967506

File: README.md (hunk 1 above)
Comment (KazuCocoa) [inline README.md:15]: Oh, it's my mistake...

Comment (hotchemi) [inline README.md:15]: Thx! Fixed.

Comment (KazuCocoa) [inline README.md:15]: :laughing:

**Pull request merged**